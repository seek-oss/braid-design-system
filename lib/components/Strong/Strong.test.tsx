import React from 'react';
import { Strong } from '..';
import { render } from 'react-testing-library';
import forEachTheme from '../../test/utils/forEachTheme';

describe('Strong', () => {
  forEachTheme(({ ThemeProvider }) => {
    test('Default', () => {
      const { container } = render(
        <ThemeProvider>
          <Strong>Children</Strong>
        </ThemeProvider>
      );
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
