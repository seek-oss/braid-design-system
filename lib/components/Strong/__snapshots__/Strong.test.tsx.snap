// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Strong Theme: jobStreet Default 1`] = `
<strong
  class="atoms__fontWeight_strong"
>
  Children
</strong>
`;

exports[`Strong Theme: seekAnz Default 1`] = `
<strong
  class="atoms__fontWeight_strong"
>
  Children
</strong>
`;

exports[`Strong Theme: seekAsia Default 1`] = `
<strong
  class="atoms__fontWeight_strong"
>
  Children
</strong>
`;

exports[`Strong Theme: wireframe Default 1`] = `
<strong
  class="atoms__fontWeight_strong"
>
  Children
</strong>
`;
